content/applications/inventory_and_mrp/repairs/repair_orders.rst

```diff
@@ -0,0 +1,248 @@
+Repair` pop-up window, and browse all products in the database.
+
+Once a :guilabel:`Product to Repair` is selected, a new :guilabel:`Product Quantity` field appears
+below it. In that field, enter the quantity (in a `0.00` format) of the product that requires
+repair.
+
+To the right of that value, click the drop-down list to select the Unit of Measure (UoM) for the
+product.
+
+In the :guilabel:`Return` field, click the drop-down menu and select the return order from which the
+product to be repaired comes from.
+
+In the :guilabel:`Under Warranty` field, tick the checkbox if the product being repaired is covered
+by a warranty. If ticked, the sales price is set to `0` for all products transferred from the repair
+order.
+
+In the :guilabel:`Scheduled Date` field, click the date to reveal a calendar popover window. From
+this calendar, select a date for the repair, and click :guilabel:`Apply`.
+
+.. image:: repair_orders/repair-orders-right-hand-form.png
+   :align: center
+   :alt: Right-hand side of blank repair order form.
+
+In the :guilabel:`Responsible` field, click the drop-down menu and select the user who should be
+responsible for the repair.
+
+In the :guilabel:`Company` field, if in a multi-company environment, select which company this |RO|
+belongs to.
+
+In the :guilabel:`Tags` field, click the drop-down menu and select which tags should be applied to
+this |RO|.
+
+Parts tab
+---------
+
+With the :guilabel:`Parts` tab selected, at the bottom of the form, click :guilabel:`Add a line`.
+Doing so creates a new line, where parts can be added, removed, or recycled.
+
+In the :guilabel:`Type` column, click the box to reveal three options to choose from:
+:guilabel:`Add` (selected by default), :guilabel:`Remove`, and :guilabel:`Recycle`.
+
+.. image:: repair_orders/repair-orders-type-column.png
+   :align: center
+   :alt: Type column options or new part under Parts tab.
+
+Choosing :guilabel:`Add` adds this part to the |RO|.
+
+Choosing :guilabel:`Remove` removes this part from the |RO|.
+
+Choosing :guilabel:`Recycle` recycles this part from the |RO|, designating it for later use or to be
+repurposed for another use in the warehouse.
+
+In the :guilabel:`Product` column, select which product (part) should be added, removed, or
+recycled. In the :guilabel:`Demand` column, change the quantity, if necessary, to indicate what
+quantity of this part should be used in the repair process.
+
+In the :guilabel:`Done` column, change the value (in a `0.00` format) once the part has been
+successfully added, removed, or recycled.
+
+In the :guilabel:`Unit of Measure` column, select the |UoM| for the part.
+
+Finally, in the :guilabel:`Used` column, tick the checkbox once the part has been used in the repair
+process.
+
+To add additional columns to the line, click the :guilabel:`optional columns drop-down toggle (two
+dots)` icon, at the far-right of the header row. Select the desired options to add to the line.
```

Comment:
I chose the term I did because it's what it read when I inspected the code, but now that I know it should be slider icon, I'll use that from now on. Suggestion implemented ✅